packages/components/src/components/widgets/WidgetsContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import { useRouter } from 'next/router';
+import { AskemFeedbackContainer, AskemProvider } from '../../components/askem';
+import useAskemContext from '../../components/askem/useAskemContext';
+import EventsCookieConsent from '../../components/eventsCookieConsent/EventsCookieConsent';
+import { useCookieConfigurationContext } from '../../cookieConfigurationProvider';
+
+type WidgetsContainerProps = {
+  appName: string;
+  hasFeedBack: boolean;
+  feedbackWithPadding: boolean;
+  consentUrl: string;
+  isModalConsent: boolean;
+  children?: React.ReactNode[];
+};
+
+export default function WidgetsContainer({
```

Comment:
How about renaming this WidgetsContainer to "UserTrackingFeatures" or something like that? Now the name feels too generic. It makes me want to use it for every widget :)

Reply:
I don't like that a "widget container" tightly couples the Askem and CookieConsent together by jsut hard coding them. The WidgetsContainerProps are just props that it's hard coded sub components are using, so should the field types be linked from the sub components?
```
type WidgetsContainerProps = {
  appName: string;
  hasFeedBack: boolean;
  feedbackWithPadding: boolean;
  consentUrl: string;
  isModalConsent: boolean;
  children?: React.ReactNode[];
};
```